src/components/application_manager/test/commands/mobile/get_dtcs_request_test.cc

```diff
@@ -54,17 +56,33 @@ namespace mobile_commands_test {
 namespace get_dtcs_request {
 
 using ::testing::_;
+using ::testing::Mock;
 using ::testing::Return;
 namespace am = ::application_manager;
 using am::commands::MessageSharedPtr;
 using am::commands::GetDTCsRequest;
 using am::event_engine::Event;
+using am::MockMessageHelper;
 namespace mobile_result = mobile_apis::Result;
 
 typedef SharedPtr<GetDTCsRequest> GetDTCsRequestPtr;
 
 class GetDTCsRequestTest
-    : public CommandRequestTest<CommandsTestMocks::kIsNice> {};
+    : public CommandRequestTest<CommandsTestMocks::kIsNice> {
+ public:
+  GetDTCsRequestTest()
+      : mock_message_helper_(*MockMessageHelper::message_helper_mock()) {}
+
+  void SetUp() OVERRIDE {
+    Mock::VerifyAndClearExpectations(&mock_message_helper_);
+  }
+  void TearDown() OVERRIDE {
+    Mock::VerifyAndClearExpectations(&mock_message_helper_);
+  }
+
+ protected:
+  am::MockMessageHelper& mock_message_helper_;
```

Comment:
@okozlovlux, MockMessageHelper already included in the current scope. There is no need to add `am::` before it.
Please, check everywhere